test-data/argoapplication.json

```diff
@@ -44,6 +44,46 @@
   "status": {
       "health": {
           "status": "Healthy"
-      }
+      },
+      "sync": {
+        "status": "Synced"
+    },
+    "resources": [
+        {
+            "kind": "Namespace",
+            "name": "bgd",
+            "version": "v1"
+        },
+        {
+            "kind": "Service",
+            "name": "bgd",
+            "version": "v1"
+        },
+        {
+            "kind": "Deployment",
+            "name": "bgd",
+            "namespace": "bgd",
+            "version": "v1",
+            "group": "apps"
+        },
+        {
+            "kind": "Route",
+            "name": "bgd",
+            "namespace": "Deployment",
+            "version": "v1",
+            "group": "route.openshift.io"
+        }
+    ],
+    "conditions": [
+        {
+            "type": "SyncError",
+            "message": "serviceaccounts is forbidden: User \"system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller\" cannot create resource \"serviceaccounts\" in API group \"\" in the namespace \"helm-nginx\""
+        },
+        {
+            "type": "InvalidSpecError",
+            "message": "invalid spec in the argocd application"
+        }
```

Comment:
Can you add an example where the message is larger than 512 characters to verify the truncation logic?